WORKSPACE

```diff
@@ -37,7 +50,10 @@ load(
 )
 
 # Download a GHC binary distribution from haskell.org and register it as a toolchain.
-rules_haskell_toolchains(version = "8.8.3")
+rules_haskell_toolchains(
+    locale = "en_US.UTF-8",
+    version = "8.8.3",
+)
```

Comment:
This part is probably not needed on Ubuntu, but was needed on my Arch box since its vanilla glibc doesn't provide a `C.UTF-8` locale.